definitions/features/disk_io.rb

```diff
@@ -0,0 +1,3 @@
+class Features::DiskIO < ForemanMaintain::Feature
```

Comment:
For this particular case, we perhaps don't need a feature at all: not all checks need a feature defined. The check itself can use the disk utility classes to do the job.